x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -117,8 +156,8 @@ features are not available while running in fips mode. The list is as follows:
 * Ingest Attachment Plugin
 * The <<certutil,`elasticsearch-certutil`>> tool. However,
  `elasticsearch-certutil` can very well be used in a non FIPS 140-2
-  enabled JVM (pointing `JAVA_HOME` environment variable to a different java
+  cofigured JVM (pointing `JAVA_HOME` environment variable to a different java
```

Comment:
> but we don't need the word anyway

I think we need it. The configuration part is important because this is not something you can get, you need to configure it to be so, so I'll be sticking with FIPS 140-2 configured JVM for this doc. Thanks for the typo spot though !